addons/api/addon/generated/models/host-catalog.js

```diff
@@ -44,6 +44,11 @@ export default class GeneratedHostCatalogModel extends BaseModel {
   })
   attributes;
 
+  @attr({
+    description: 'Secrets specific to the plugin type',
+  })
+  secrets;
```

Comment:
This is like `attributes` in that it's handled transparently by the serializer.  We don't need to declare it in the model.